options/locale/locale_en-US.ini

```diff
@@ -86,6 +86,8 @@ loading = Loading…
 
 error404 = The page you are trying to reach either <strong>does not exist</strong> or <strong>you are not authorized</strong> to view it.
 
+api_to_home_page = Return Home Page
```

Comment:
```suggestion
api_to_home_page = Return to Home Page
```

Reply:
This could also maybe be a more generic translation in this section instead of API specific:

https://github.com/go-gitea/gitea/blob/master/options/locale/locale_en-US.ini#L1

Reply:
> This could also maybe be a more generic translation in this section instead of API specific:
> 
> https://github.com/go-gitea/gitea/blob/master/options/locale/locale_en-US.ini#L1

Hmm, I think so